test/images/Makefile

```diff
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include  ../../hack/make-rules/Makefile.manifest
```

Comment:
- we don't use any rules from this
- if you inspect the targets we do have, we're using `docker manifest ...`, not `manifest-tool`